spec/docile_spec.rb

```diff
@@ -414,6 +414,12 @@ def dsl_eval_string(string)
         expect(dsl.foo).to eq(0)
         expect(dsl.bar).to eq(1)
       end
+
+      context "when the DSL object is frozen" do
+        it "can call non-mutative code without raising an exception" do
+          expect { dsl.freeze.dsl_eval_string('1 + 2') }.not_to raise_error
```

Comment:
Hi @tomeon ,
I think the `dsl` object is not frozen so you need to freeze the `dsl` object before evaluating the example which you added.